### DISCUSSION TITLE
Switch between windows

### DISCUSSION THREAD

**Discussion by Volta73** [rxhanson/Rectangle#464] [Q&A] [answered]
Just recently switched to Rectangle and really happy with it so far. Thanks a lot to all who are involed in development!

One shortcut I used with BetterTouchTool before was switching between windows (e.g. with Preview). Is there a way this is currently available with Rectangle or would this maybe something to look into in future versions? The Mac OS native command is just annoying with german QWERTZ layout... Thanks again!

**Comment by rxhanson** [comment, marked as answer]
I think you're talking about **cmd + `**

My recommendation is just to remap it in macOS to whatever you want. 
![s](https://user-images.githubusercontent.com/13651296/118258565-50eec080-b4b0-11eb-8d35-96028d160378.png)
